openpype/hosts/maya/api/lib.py

```diff
@@ -2097,15 +2106,25 @@ def reset_frame_range():
     if handle_end is None:
         handle_end = handles
 
-    frame_start -= int(handle_start)
-    frame_end += int(handle_end)
+    if task_name in with_handles:
```

Comment:
I actually suggest boolean type to enable/disable the setting. It's not the task-types-enum type. 
I agree that it is getting much more confused for the way it does to the frameHandle(the initial purposes seems to include the frame handle in timeline while the function actually excludes the handles)... and we must deal with the setting more carefully if we want to continue to make it as a part of the setting.

Reply:
Yeah, I agree that the naming is a bit confusing atm. How about `Use handles on timeline`?

Also agree that it should be a bit easier to use for all task types. How about the setting is a boolean (off by default)?
When enabled the functionality would be enabled for all tasks on the project. Then have a `Limit to Task Types` option, if a studio wanted to limit to a handful of task types.

Reply:
I think @friquette you wanted this enabled always and others mentioned they preferred the old behavior? If so I maybe even wouldn't make it task dependent to begin with and just a project-wide setting? We can always add task overrides if people need it and have a use case for it later? Or do we really think people will end up customizing this per task?

Also, use handles on timeline still sounds confusing. The old behavior did include the handles too. It just wouldn't exclude the handles in the active timeline play area.
According to this [Maya documentation on the Range slider](
https://help.autodesk.com/view/MAYAUL/2022/ENU/index.html?guid=GUID-E9E054A1-A5D0-412C-976A-F57C0F6960B1) we can use the terminology Playback range versus Animation range. As such I would say the toggle does: *"Exclude handles from playback range** or **Set playback range to exclude handles**.

Reply:
> Set playback range to exclude handles

+1

> We can always add task overrides if people need it and have a use case for it later? Or do we really think people will end up customizing this per task?

Yup, think that is fair. Boolean type for project wide toggle. If another studio needs more granular control, then it can be implemented later.